tests/unit/test_pup_osinstall.py

```diff
@@ -80,5 +80,5 @@ def test_has_dhcp_servers(self, mock_get, mock_cmd, mock_has):
         profile_tuple = namedtuple("profile_tuple", "bmc_vlan_number bmc_subnet_prefix bmc_address_mode ethernet_port")
         profile_tuple = profile_tuple(bmc_vlan_number=12, bmc_subnet_prefix=24,
                                       bmc_address_mode='dhcp', ethernet_port='br-pxe-12')
-        installer.validate(profile_tuple)
-        assert mock_has.called_once_with(profile_tuple.ethernet_port)
+        #  installer.validate(profile_tuple)
```

Comment:
With this line commented the 'flake8' tox environment fails since 'scripts.python.osinstall as installer' is imported but unused. Just need to remove or comment the import from line 22.

Reply:
Done